tests/test_itemized.py

```diff
@@ -173,6 +171,14 @@ def test_filter_line_number(self):
         results = self._results(api.url_for(ScheduleBView, line_number='f3X-21', **self.kwargs))
         self.assertEqual(len(results), 1)
 
+    def test_sched_b_spender_committee_type_filter(self):
```

Comment:
One small thing - would this test be more meaningful if we added another `spender_committee_type` that's not `S` so we can be sure it's filtering properly?

Reply:
yes. good idea.Thanks.